packages/example/src/1week/04/__submit__/minsoo-web.ts

```diff
@@ -1,13 +1,15 @@
-export function convertToConditionalUpperCase(words: string[]): string[] {
-  const capitalized = []
-
-  for (let i = 0; i < words.length; i += 1) {
-    if (words[i].length > 5) {
-      capitalized.push(words[i].toUpperCase())
-    } else {
-      capitalized.push(words[i].toLowerCase())
-    }
-  }
+const condition = (text: string, criteria = 5) => text.length > criteria
```

Comment:
condition 함수의 이름이 조금 추상적으로 느껴져서

```suggestion
const isLongerThanCriteria = (text: string, criteria = 5) => text.length > criteria
```

`is` 프리픽스로 `boolean`임을 더욱 분명하게 알려주는 구체적인 네이밍 `isLongerThanCriteria`도 좋을 것 같아요

Reply:
아 너무 명확하네요 ㅠㅠ 좋습니다 충일님!! 반영하겠습니다 :)